acceptance/services_test.go

```diff
@@ -0,0 +1,42 @@
+package acceptance_test
+
+import (
+	"github.com/epinio/epinio/acceptance/helpers/catalog"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Services", func() {
+	Describe("delete services", func() {
+		var namespace, service string
+
+		BeforeEach(func() {
+			namespace = catalog.NewNamespaceName()
+			env.SetupAndTargetNamespace(namespace)
+
+			service = catalog.NewServiceName()
+
+			out, err := env.Epinio("", "service", "create", "mysql-dev", service)
+			Expect(err).ToNot(HaveOccurred(), out)
+
+			out, err = env.Epinio("", "service", "show", service)
+			Expect(err).ToNot(HaveOccurred(), out)
+			Expect(out).To(MatchRegexp("Name.*|.*service-instance-268892044"))
```

Comment:
Does it work with the "fixed" name?

Reply:
Stupid me. It only worked because the `|` needs to be escaped (otherwise it's a regex "or"). Fixing